anchore_engine/services/apiext/api/helpers/image_content_response.py

```diff
@@ -144,6 +177,9 @@ def _build_java_response(content_data):
             el["maven-version"] = content_data[package]["maven-version"]
             el["origin"] = content_data[package]["origin"] or "Unknown"
             el["cpes"] = content_data[package].get("cpes", [])
+            el["metadata"] = _transform_java_pom_metadata(
```

Comment:
This transformation should occur in the grype mapper, not in the image content builder. Image content response should be provider unaware, it shouldn't really worry about formatting content based on the vulnerabilities provider.